mongodbatlas/data_source_mongodbatlas_org_invitation.go

```diff
@@ -0,0 +1,97 @@
+package mongodbatlas
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceMongoDBAtlasOrgInvitation() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceMongoDBAtlasOrgInvitationRead,
+		Schema: map[string]*schema.Schema{
+			"org_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"username": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"invitation_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"expires_at": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"created_at": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"inviter_username": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"roles": {
+				Type:     schema.TypeSet,
+				Optional: true,
```

Comment:
It should be computed, isn't it? or Why is optional?

Reply:
Changed